ocfweb/docs/docs/services/web/jekyll.md

```diff
@@ -0,0 +1,83 @@
+[[!meta title="Jekyll"]]
+
+[Jekyll](https://jekyllrb.com) is a popular static website generator.
+
+## Set up
+
+1. Go to our [web-based SSH client](https://ssh.ocf.berkeley.edu/) and sign in
+   with your username and password. Or, in terminal, [[SSH|doc services/shell]]
+   into `ssh.ocf.berkeley.edu`.
+
+2. Set up [rbenv](https://github.com/rbenv/rbenv):
+   ```shell
+   git clone https://github.com/rbenv/rbenv.git ~/.rbenv
+   echo 'export PATH="~/.rbenv/bin:$PATH"' >> ~/.bash_profile
+   echo 'eval "$(rbenv init -)"' >> ~/.bash_profile
+   source ~/.bash_profile
+   mkdir -p "$(rbenv root)"/plugins
+   git clone https://github.com/rbenv/ruby-build.git "$(rbenv root)"/plugins/ruby-build
+   ```
+
+3. Create and enter project directory "Jekyll_project" (can replace
+   "Jekyll_project" with any name you'd like):
+   ```shell
+   mkdir Jekyll_project
+   cd Jekyll_project
+   ```
+
+4. Configure ruby via [rbenv](https://github.com/rbenv/rbenv), then install
+   jekyll:
+   ```shell
+   rbenv install -l | grep -v - | tail -1 | xargs rbenv install
+   rbenv install -l | grep -v - | tail -1 | xargs rbenv local
+   rbenv rehash
+   gem install jekyll bundler
+   ```
+
+## Create a new (template) site
+
+1. In project directory ("Jekyll_project"):
```

Comment:
You might want to consider more explicitly stating to cd into Jekyll_project. From experience, people may otherwise miss the significance of this line.